### ISSUE TITLE
Chat UI messes up when zooming the page and resetting it again

### ISSUE THREAD

**Issue by exaby73** [benawad/dogehouse#2512]
Chat UI messes up when zooming the page and resetting it again

1. Zoom the page using CTRL + + or CMD + +
2. Reset zoom with CTRL + 0 or CMD + 0

**Expected behavior**
UI should not change

**Screenshots**
![image](https://user-images.githubusercontent.com/50555895/116839493-3be07a80-abf0-11eb-80dd-bcf7f9c33dcd.png)

**What device are you on?**
Brave Browser (Kali Linux)

**Comment by Isaiah-Hamilton** [comment]
Just to add to this bug I get the same problem but the chat messages are more spread out. I Did the same steps as you did to recreate to bug.

https://user-images.githubusercontent.com/70828596/116916438-4c5a1a80-ac1b-11eb-9dd6-6d9472cdf6bf.mov

**Steps to reproduce the behavior:**

Zoom the page using CTRL + + or CMD + +
Reset zoom with CTRL + 0 or CMD + 0

**Expected behavior**
UI should not change

**What device are you on?**
Brave Browser (MacOS)

**Comment by exaby73** [comment]
@Isaiah-Hamilton Yes I have noticed that as well. Not sure what causes it to sometimes spread out or become squished like in my screen shot. Also, I have noticed that it resets back to normal upon a new message. But that's probably react re-building that component
